libmalino-cs/MGS/Framebuffer.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.Drawing;
+
+
+namespace libmalino.FirstMalino;
```

Comment:
Also I think this namespace is incorrect here.

Reply:
srry i forgot to change it lol

Reply:
Then change it.